dist/net.unvanquished.Unvanquished.appstream.xml

```diff
@@ -0,0 +1,85 @@
+      from those two-legged invaders who only know how to turn
+      beautiful organic equilibrium into lifeless machines and dead
+      concreted worlds.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">net.unvanquished.Unvanquished.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/04/perseus_tyrant.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-082658-002.unvanquished-0.52-human-builder.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-082658-002.unvanquished-0.52-human-builder-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2020/09/20200909-025733-001.unvanquished-rocket-missile.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2020/09/20200909-025733-001.unvanquished-rocket-missile-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-081116-002.unvanquished-0.52-alien-team.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-081116-002.unvanquished-0.52-alien-team-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2020/09/20200909-001900-000.unvanquished-lasgun-weapon.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2020/09/20200909-001900-000.unvanquished-lasgun-weapon-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-083923-002.unvanquished-0.52-alien-builder.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-083923-002.unvanquished-0.52-alien-builder-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-072808-002.unvanquished-0.52-human-team.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-072808-002.unvanquished-0.52-human-team-768x432.jpg</image>
+    </screenshot>
+  </screenshots>
+
+  <url type="homepage">https://unvanquished.net</url>
+  <url type="bugtracker">https://github.com/Unvanquished/Unvanquished/issues</url>
+  <url type="help">https://wiki.unvanquished.net/wiki/Main_Page</url>
+
+  <releases>
+    <release version="0.52.0" date="2021-05-14">
+      <description>
+        <p>
+          The first beta release, coming after years of continuous work
```

Comment:
Does there really need to be a release-specific description or URL? It seems annoying to have to update them.

Reply:
It needs not, but then you loose the version information in the store, and the mini changelog

Reply:
It's nice to have them because they act like a changelog but shown in stores. If Unvanquished had a changelog/NEWS file somewhere, then `appstream-util news-to-appdata` could have automated the process of updating the releases shown in this appstream file.